gluonnlp/model/train/embedding.py

```diff
@@ -199,14 +220,14 @@ class FasttextEmbeddingModel(EmbeddingModel, HybridBlock):
     ----------
     token_to_idx : dict
         token_to_idx mapping of the vocabulary that this model is to be trained
-        with. token_to_idx is used when to_token_embedding is called. For
+        with. token_to_idx is used for __getitem__ and __contains__. For
```

Comment:
Minor: the token_to_idx contains the n-gram and the special word, probably name it as `ngram_to_idx`?

Reply:
The `subword_function` argument a few lines below is used to map words to the list of hashed ngrams. The  token_to_idx is needed because for tokens observed during training, an extra vector is learned. So when __getitem__([tokens]) is called, for each token I need  to check if it was known during training (using token_to_idx). I'll try to make this  clearer in the description.